docs/environments/environments.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Environments"
+---
+
+# Available environments
+
+| Env                                                                                                                                                      | Obs/Action spaces                   | Objectives                                                    | Description                                                                                                                                                                                                                                                                                                |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `deep-sea-treasure-v0` <br><img src="https://raw.githubusercontent.com/LucasAlegre/mo-gym/main/screenshots/dst.png" width="200px">                       | Discrete / Discrete                 | `[treasure, time_penalty]`                                    | Agent is a submarine that must collect a treasure while taking into account a time penalty. Treasures values taken from [Yang et al. 2019](https://arxiv.org/pdf/1908.08342.pdf).                                                                                                                          |
```

Comment:
Update paths to: `https://raw.githubusercontent.com/Farama-Foundation/MO-Gymnasium/main/screenshots/*.png`